src/components/modal/Modal.jsx

```diff
@@ -0,0 +1,71 @@
+import { NavLink } from "react-router-dom";
+
+import PropTypes from "prop-types";
+
+import iconClose from "../../images/closeIcon.svg";
+import search from "../../images/search.svg";
+
+import styles from "./Modal.module.scss";
+
+const Modal = ({ setIsOpenModal }) => {
+  return (
+    <div className={styles.modal}>
+      <div className={styles.modal_content}>
+        <button
+          type="button"
+          className={styles.btn_close}
+          onClick={() => setIsOpenModal(false)}
```

Comment:
same perfomance issue. 

better to throw in prop onCloseModal 
 onClick={onCloseModal}